pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -359,9 +359,9 @@
   farsi_page_founders_slide_06: "In 2013, Hadi and Ali launched the education nonprofit Code.org, which Hadi continues to lead as CEO. Since its inception, Code.org has started a massive movement to teach computer science and coding to K-12 students globally. It is also the largest provider of computer  science curricula in the world, with over 80 million student users in 180 countries."
   farsi_page_support_heading: "Your Support and Your Voice"
   farsi_page_support_heading_02: "Thank you for your interest in Code.org in Farsi."
-  farsi_page_support_get_updates: "Sign up below to learn more about this specific project and get updated on its staged release dates."
+  farsi_page_support_get_updates: "[Sign up](%{form_url}) to learn more about this specific project and get updated on its staged release dates."
```

Comment:
Why are there brackets in the [Sign Up] text?

Reply:
This is how you make a hyperlink within a string — I think it's using Markdown.